### DISCUSSION TITLE
What happened to TypedPropertyRector?

### DISCUSSION THREAD

**Discussion by tyteen4a03** [rectorphp/rector#7864] [Q&A]
I would like to convert `@var` annotations on class properties to typed properties, but it looks like `TypedPropertyRector` is no longer a rule.

How do I do this now?

**Comment by TomasVotruba** [comment]
This rule was dangerous and create many false positives based on invalid docblocks.
It was split into many smaller specific rules based on strict type declarations :+1: 

See https://getrector.com/blog/new-in-rector-015-complete-safe-and-known-type-declarations
